tools/travis-build-and-push-docker.sh

```diff
@@ -21,23 +19,30 @@ export BUILDS=`pwd`
 export MEMBER_TGZ=mongooseim.tar.gz
 
 DOCKERHUB_TAG=${TRAVIS_BRANCH}
+VERSION=`tools/generate_vsn.sh`
+GIT_REF=`git rev-parse --short HEAD`
 
 if [ ${TRAVIS_PULL_REQUEST} != 'false' ]; then
     DOCKERHUB_TAG="PR-${TRAVIS_PULL_REQUEST}"
 elif [ ${TRAVIS_BRANCH} == 'master' ]; then
     DOCKERHUB_TAG="latest";
 fi
 
+IMAGE_TAG=${DOCKERHUB_USER}/mongooseim:${DOCKERHUB_TAG}
+
 git clone https://github.com/esl/mongooseim-docker.git
 cd mongooseim-docker
+git checkout 843558f
```

Comment:
Thanks for the suggestion @mentels. How and where would you document it? I though it'd be enough to just add

>frozen version of esl/mongooseim-docker

To the PR description.